Frontend/src/app/views/dashboard/project-details/project-details.component.scss

```diff
@@ -58,7 +58,6 @@ p, nb-tag-list {
 
 @media only screen and (max-width: 1000px) {
     .container {
-        padding-left: 5%;
-        padding-right: 5%;
+        padding: 40px;
     }
```

Comment:
Can probably just remove this and use the padding that the shared `.container` gives